osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/mapping/DataObjectToEventListConverter.java

```diff
@@ -86,8 +109,19 @@ private DateTime extractDateTime(final List<DataObject> eventData) throws Protoc
         return dateTime;
     }
 
-    private Short extractCode(final List<DataObject> eventData) throws ProtocolAdapterException {
+    private Long extractEventDuration(EventLogCategoryDto eventLogCategory, List<DataObject> eventData)
+            throws ProtocolAdapterException {
+        if (!eventData.get(1).isNumber()) {
+            throw new ProtocolAdapterException("eventData value is not a number");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "eventData value is not a number" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1192)